verification/curator-service/ui/src/components/AutomatedSourceForm.tsx

```diff
@@ -181,6 +200,29 @@ export default function AutomatedSourceForm(props: Props): JSX.Element {
                                         required
                                     />
                                 </div>
+                                <div className={classes.formSection}>
+                                    <ChipInput
+                                        classes={{
+                                            helperText: classes.errorHelper,
+                                        }}
+                                        data-testid="recipients"
+                                        helperText={
+                                            errors.notificationRecipients
+                                                ? 'Values must be valid email addresses'
+                                                : undefined
+                                        }
+                                        fullWidth
+                                        alwaysShowPlaceholder
+                                        placeholder="Notification recipient emails"
+                                        defaultValue={[props.user.email]}
```

Comment:
Would be better to have this as initialValues.notificationRecipients so it's only defined in one place

Reply:
SG, done.